aspnetcore/tutorials/first-mvc-app/start-mvc.md

```diff
@@ -30,19 +29,21 @@ At the end of the series, you'll have an app that manages and displays movie dat
 
 [View or download sample code](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/tutorials/first-mvc-app/start-mvc/sample) ([how to download](xref:index#how-to-download-a-sample)).
 
+At the end of the series, you'll have an app that manages, validates and displays movie data. 
+
+![Completed application showing the search UI](~/tutorials/first-mvc-app/search/_static/9/search2_90.png)
+
+![Completed application showing client side valiation features](~/tutorials/first-mvc-app/validation/_static/9/val90.png)
+
 ## Prerequisites
 
 # [Visual Studio](#tab/visual-studio)
 
-[!INCLUDE[](~/includes/net-prereqs-vs-8.0.md)]
+[!INCLUDE[](~/includes/net-prereqs-vs-9.0.md)]
 
 # [Visual Studio Code](#tab/visual-studio-code)
 
-[!INCLUDE[](~/includes/net-prereqs-vsc-8.0.md)]
-
-# [Visual Studio for Mac](#tab/visual-studio-mac)
-
-[!INCLUDE[](~/includes/net-prereqs-mac-8.0.md)]
+[!INCLUDE[](~/includes/net-prereqs-vsc-9.0.md)]
```

Comment:
Have you verified that the instructions work with C# Dev Kit?

Reply:
Good point. I did not.  It was blocking the prompt for creating assets and could not be used as of late last year per my notes.  I will test the latest with C# Dev Kit reinstalled.

Reply:
Tested now:  Yes it does work.  Assets are just added without the dialog that used to stop the works and then keep the add assets UI from showing.  Looks like it was also added back into the prereqs already.  So nothing more to change.  Thanks for bringing it up.